test/test_oneview_storage_volume_attachment.py

```diff
@@ -72,128 +71,54 @@
 }
 
 
-class StorageVolumeAttachmentClientConfigurationSpec(unittest.TestCase):
-    @mock.patch.object(OneViewClient, 'from_json_file')
-    @mock.patch.object(OneViewClient, 'from_environment_variables')
-    @mock.patch('oneview_storage_volume_attachment.AnsibleModule')
-    def test_should_load_config_from_file(self, mock_ansible_module, mock_ov_client_from_env_vars,
-                                          mock_ov_client_from_json_file):
-        mock_ov_instance = mock.Mock()
-        mock_ov_client_from_json_file.return_value = mock_ov_instance
-        mock_ansible_instance = create_ansible_mock({'config': 'config.json'})
-        mock_ansible_module.return_value = mock_ansible_instance
+class StorageVolumeAttachmentSpec(ModuleContructorTestCase,
+                                  ErrorHandlingTestCase):
```

Comment:
The tests of this class aren't been running, missing extend  `unittest.TestCase`